### PR TITLE
Update people-example.md

### PR DESCRIPTION
**Pull request by polio77**
Adding section explaining how 'Working With' is implemented. Requires upload of working-with.png shared
working-with

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[e39840e](https://github.com/polio77/microsoft-graph-docs/pull/3621/commits/e39840e49cb67cafdcc56d0e85328f5ce151efc1)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/people-example.md](https://github.com/polio77/microsoft-graph-docs/blob/patch-1/concepts/people-example.md) | :warning:Warning | | [Details](#user-content-concepts%2Fpeople-example.md)

<a name="concepts%2Fpeople-example.md"></a>
### [concepts/people-example.md](https://github.com/polio77/microsoft-graph-docs/blob/patch-1/concepts/people-example.md)
 - **Line 945**: **[Warning]** Invalid file link:(~/images/working-with.png).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C13%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812131410199408-3621%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=BsSE5mjFdVjSFJutRx10rgIWqoxPKPPSg3uMVtwjWb8%3D&st=2018-12-13T14%3A08%3A27Z&se=2019-01-13T14%3A13%3A27Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: concepts/people-example.md [hunk 1]
```diff
@@ -939,3 +939,33 @@ GET https://graph.microsoft.com/v1.0/me/people?$search="tylerle@example.com"  //
...
 ```
+
+## Delve “Working With” feature implementation
+ 
+![Image of Working With relationships](images/working-with.png "Image of Working With relationships")
+
+**There must be a public relationship between the profile owner and the other people in order for those people to show up on the profile owner's list**
+ 
+Examples of public relationships:
+* They are connected in the org chart: Manager, Direct, Peers (share the same manager)
```
Comment (jasonjoh) [inline concepts/people-example.md]: Does "Direct" mean direct report? If so I'd change it to "direct report" to be clear. Also I'd uncapitalize these.

File: concepts/people-example.md [hunk 2]
```diff
@@ -939,3 +939,33 @@ GET https://graph.microsoft.com/v1.0/me/people?$search="tylerle@example.com"  //
...
+ 
+Examples of private communication:
+* They send emails to each other where the name of the other person is in the TO line
+* They are invited to meetings using the name of the other person on the calendar invite 
+ 
+Examples of public interaction:
+* They send or receive emails to/from each other as part of public group 
+* They are invited to meetings as part of group, or where more than X people are invited
+ 
+The ranking doesn’t change based on who User A is (the person looking at someone else's page). The ranking is determined by the interaction level between User B (profile owner) and User C (person showing up on profile owner's list).
```
Comment (jasonjoh) [inline concepts/people-example.md]: Another "fancy" quote, this time a single-quote. Needs to be replaced with `'`.

**Comment by jasonjoh** [comment]
Redirecting this PR into a temporary branch. This will allow us to stage the changes before we merge into master.

**Pull request closed without merging**